docs/dev-omnistudio.md

```diff
@@ -0,0 +1,47 @@
+# Develop a Project that uses Omnistudio
```

Comment:
I think the official spelling is `OmniStudio`, although our own website is not consistent.